tests/unit/tf/examples/test_usecase_ecommerce_session_based.py

```diff
@@ -59,3 +59,16 @@ def test_usecase_ecommerce_session_based(tb):
             "regularization_loss",
         ]
     )
+    metrics_transformer = tb.ref("metrics_transformer")
+    assert set(metrics_transformer.keys()) == set(
```

Comment:
Can you add `pytest.importorskip("transformers")` to the top of this file to reflect the dependency on transformers now

Reply:
yes, will do. thanks!